website/integrations/services/glitchtip/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Glitchtip
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Glitchtip
+
+> Bugs are inevitable in web development. The important thing is to catch them when they appear. With GlitchTip, you can rest easy knowing that if your web app throws an error or goes down, you will be notified immediately. GlitchTip combines error tracking and uptime monitoring in one open-source package to keep you and your team fully up-to-date on the status of your projects.
+>
+> -- https://glitchtip.com/documentation
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `glitchtip.company` is the FQDN of the Glitchtip install.
+-   `authentik.company` is the FQDN of the authentik install.
+
+## authentik Configuration
+
+Create an OAuth2/OpenID provider with the following parameters:
+
+-   Client Type: `Confidential`
+-   Redirect URIs: `https://glitchtip.company/auth/authentik`
+
+Note the Client ID and Client Secret values.
+
+Create an application, using the provider you've created above. Note the slug of the application you've created.
+
+## Glitchtip Configuration
+
+Configuration of OpenID Connect providers in Glitchtip is done using Django Admin.
+
+Create a superuser:
```

Comment:
```suggestion
1. Create a superuser:
```

Reply:
Do think we need to be more specific about where one does this? Is in the UI of the Django Admin? If Yes, you could say: 
1. In the Django Admin UI, create a superuser.